lib/cli/commands/listpairs.ts

```diff
@@ -1,11 +1,34 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPairsRequest } from '../../proto/xudrpc_pb';
+import { ListPairsRequest, ListPairsResponse } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { getPairCurrencies } from '../../utils/utils';
+
+const HEADERS = [colors.blue('PairId')];
```

Comment:
One more thing, `PairId` is something of an internal term. For end-user purposes I think we can call this `Trading Pair` or something more descriptive like that.

Reply:
Agree! thanks for the tip.